libshaderc_util/src/spirv_tools_wrapper.cc

```diff
@@ -36,13 +38,25 @@ void SpirvToolsOutputDiagnostic(spv_diagnostic diagnostic, std::string* dest) {
   *dest = os.str();
 }
 
-}  // anonymous namespace
+// Gets the corresponding target environment used in SPIRV-Tools.
+spv_target_env GetSpirvToolsTargetEnv(Compiler::TargetEnv env) {
+  switch (env) {
+    case Compiler::TargetEnv::Vulkan:
+      return SPV_ENV_VULKAN_1_0;
+    case Compiler::TargetEnv::OpenGL:
+      return SPV_ENV_OPENGL_4_5;
+    case Compiler::TargetEnv::OpenGLCompat:
+      return SPV_ENV_OPENGL_4_5;  // TODO(antiagainst): is this correct?
```

Comment:
I'm not 100% sure about this, but let's go with it.  I don't think that OpenGL restricted what versions could be used with compatibility profile.